tgui/packages/tgui/interfaces/DnaConsole.js

```diff
@@ -653,6 +653,14 @@ const MutationInfo = (props, context) => {
                 is_activator: 0,
                 source: mutation.Source,
               })} />
+            <Button
+              icon="syringe"
+              disabled={!mutation.Active}
```

Comment:
Not sure how to check for that in TGUI - if you had some resources I could look over I'm happy to do my own reading otherwise I'll tinker tonight to figure it out